### DISCUSSION TITLE
Trend Damping with AutoETS

### DISCUSSION THREAD

**Discussion by tbellamey** [Nixtla/statsforecast#725] [Q&A]
Is it possible to specify the smoothing  parameter for trend damping (i.e. phi) when fitting & forecasting with AutoETS? For example, if I wanted to manually specify a stronger or weaker damping effect vs. letting the model infer that parameter when damped=True?

**Comment by jmoralez** [comment]
Hey. It's not currently possible but it's straightforward to implement, I've opened #727 to track it.
